### PR TITLE
Adding scope to escrow

### PR DESCRIPTION
**Pull request by dallasjohnson**

### CODE REVIEW

File: contracts/dacescrow/dacescrow.cpp [hunk 1]
```diff
@@ -19,12 +20,14 @@ namespace eosdac {
         }
 
         require_auth(from);
+        auto tokens = split(memo, ":");
+        check(tokens.size() == 3, "Invalid memo format");
```
Comment (angelol) [inline contracts/dacescrow/dacescrow.cpp:24]: I've added this additional check to prevent out-of-bounds array access.

**Review by angelol — APPROVED** [review]
LGTM!

**Review by dallasjohnson — COMMENTED** [review]
LGTM 👍

**Review by dallasjohnson — COMMENTED** [review]
LGTM 👍

**Pull request merged**